invenio_communities/serializers/schemas/community.py

```diff
@@ -38,6 +54,8 @@ class CommunitySchemaV1(Schema):
     page = fields.String()
     curation_policy = fields.String()
     last_record_accepted = fields.DateTime()
+    links = fields.Function(
+            serialize=lambda c: LinksSchemaV1().dump(c)[0])
```

Comment:
There was no way to make it works with ``fields.Nested(LinksSchemaV1)``, for some reason it was skipping the ``links`` field completely.